tests/cli_test_case.py

```diff
@@ -187,6 +187,9 @@ def cli(self, *args, **kwargs) -> click.testing.Result:
         """
         return CliRunner(mix_stderr=mix_stderr).invoke(main, args, catch_exceptions=False, **kwargs)
 
+    def assert_success(self, result: click.testing.Result):
+        self.assertEqual(result.exit_code, 0, result.stdout)
```

Comment:
Makes it easier to diagnose a problem when the command unexpectedly fails. I'll come back and retrofit existing assertions with this one later.